sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryDirectReadSchemaTransformProvider.java

```diff
@@ -123,15 +124,22 @@ public static Builder builder() {
           .Builder();
     }
 
+    @SchemaFieldDescription("The SQL query to be executed to read from the BigQuery table.")
     @Nullable
     public abstract String getQuery();
 
+    @SchemaFieldDescription(
+        "The BigQuery table to read from. Format: [${PROJECT}:]${DATASET}.${TABLE}")
     @Nullable
     public abstract String getTableSpec();
 
+    @SchemaFieldDescription(
+        "The Google standard SQL expression to read only the rows matched from the specified filter. If no value is specified, then all rows are returned.")
     @Nullable
     public abstract String getRowRestriction();
 
+    @SchemaFieldDescription(
+        "Read only the specified fields (columns) from a BigQuery table. Fields may not be returned in the order specified. If no value is specified, then all fields are returned.")
     @Nullable
     public abstract List<String> getSelectedFields();
```

Comment:
How does Syndeo allow users to input a list? should that be mentioned in the description?

Reply:
done (seems like it's using commas to separate each entry within a list)